lively.morphic/events/KeyHandler.js

```diff
@@ -203,8 +203,8 @@ export default class KeyHandler {
       .replace(regexps.tab, '⇥')
       .replace(regexps.enter, '⏎')
       .replace(regexps.shift, '⇧')
-      .replace(regexps.backspace, '⌫')
-      .replace(regexps.delete, '⌦')
+      .replace(regexps.backspace, bowserOS() == 'mac' ? '⌫' : 'Backspace')
+      .replace(regexps.delete, bowserOS() == 'mac' ? '⌦' : 'Del')
```

Comment:
`Del` seems reasonable, but why would you change backspace as well? All keyboards I ever had had the correct symbol on it?

Reply:
Okay, was not so familiar with anything different to mac. Will change that back.